Libplanet.Store/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Libplanet.Extensions.Cocona.Tests")]
+[assembly: InternalsVisibleTo("Libplanet.Tests")]
```

Comment:
Hopefully we can get rid of this at some point. 😥

Reply:
https://github.com/planetarium/libplanet/pull/3303/commits/4c952a55214451fc516f76c28d8a498329ead74c Amended cc. @OnedgeLee